FFmpeg.AutoGen.CppSharpUnsafeGenerator/Definitions/StructureField.cs

```diff
@@ -5,5 +5,7 @@ internal class StructureField : ICanGenerateXmlDoc
         public string Name { get; set; }
         public TypeDefinition FieldType { get; set; }
         public string Content { get; set; }
+        public bool IsDeprecated { get; set; }
```

Comment:
Must be named IsObsolete as it was already done in other structure.

Reply:
Done.